packages/compass/src/app/utils/telemetry.ts

```diff
@@ -200,6 +203,52 @@ export function trackNewConnectionEvent(
   }
 }
 
+export function trackConnectionDisconnectedEvent(
```

Comment:
I did miss it when we moved connection attempt etc here, but i don't understand why we are moving these here. 

I would love if we have all of the domain specific `track` functions in one place, i don't think this is what is happening though. Why do we have this special "place" for connection events only? And if this is a road that we want to pursue, how do we make sure is consistent everywhere and that we don't have 5 different ways to do the same thing?

Reply:
Answered in https://github.com/mongodb-js/compass/pull/6018#discussion_r1677948900 (I think those basically express the same concerns)